R/retrieveData-KNMI-OpenAPI.R

```diff
@@ -0,0 +1,218 @@
+# KNMI Open API ####
+
+# Author: Cherine Jantzen
+# Created: 2024/02/20
+# Last updated: 2024/02/22
+
+
+# I. Load packages --------------------------------------------------------
+
+library(dplyr)
+library(stringr)
+library(lubridate)
+library(httr)
+library(purrr)
+library(jsonlite)
+library(here)
+
+
+# Function to retrieve netCDF files from KNMI Open API --------------------
+
+# Arguments:
+# - variable: Character providing the weather variable of interest. Either "mean temperature", "max temperature", "min temperature" or "precipitation".
+# - start_date: Character providing the start date of period for which to retrieve data in the format: "yyyy-mm-dd".
+# - end_date: Character providing the end date of period for which to retrieve data in the format: "yyyy-mm-dd". If empty, current year is used.
+# - season_start: Vector specifying the start month and day of the period of the year for which data should be retrieved in the format: c(mm, dd), e.g., c(2, 10) for February 10. Allows to limit data retrieval to a selected period of the year. If empty, data for the full year is retrieved. 
+# - season_end: Vector specifying the end month and day of the period of the year for which data should be retrieved in the format: c(mm, dd), e.g., c(5, 15) for May 15. If provided, season_start also needs to be provided.
+# - version: Character specifying the version of the data set. (Version "5" for mean temperature and precipitation, version "2" for min and max temperature)
+# - knmi_open_key: Character specifying user-specific KNMI Open API key (request here: https://developer.dataplatform.knmi.nl/apis).
+
+
+call_knmi_open_api <- function(variable = c("mean temperature", "max temperature",
```

Comment:
The EDR function was called `request_...()`. I don't mind either "request" or "call", but it would be good to be consistent, I think.